backend/EasySpeak.Core/EasySpeak.Core.BLL/Services/UserService.cs

```diff
@@ -94,6 +115,13 @@ public async Task<List<UserShortInfoDto>> GetFilteredUsers(UserFilterDto userFil
         {
             filteredUsers = filteredUsers.Where(u => u.Tags.Any(t => filter.Topics.Contains(t.Name)));
         }
+        if (filter.Compatibility is not null)
+        {
+            var compatibleUsers =
+                await GetRecommendedUsers((int)filter.Compatibility, filteredUsers.Select(x => x.Id).ToList());
+            filteredUsers = filteredUsers.Where(x => compatibleUsers.Contains(x.Id));
+        }
```

Comment:
Have you set some value to recomendation to show it on FE?

Reply:
Yes, It works, but I forgot to make screenshot, I will do it with requested changes